cocotb/binary.py

```diff
@@ -376,16 +406,25 @@ def set_binstr(self, string):
                                  (char, self.__class__.__name__))
         self._str = string
         self._adjust()
-
-    binstr = property(get_binstr, set_binstr, None,
-                      "Access to the binary string.")
+        
+    @property
+    def binstr(self):
+        "Access to the binary string."
+        return self.get_binstr()
+        
+    @binstr.setter
+    def binstr(self, val):
+        "Access to the binary string."
+        self.set_binstr(val)
 
     def _get_n_bits(self):
         """The number of bits of the binary value."""
         return self._n_bits
-
-    n_bits = property(_get_n_bits, None, None,
-                      "Access to the number of bits of the binary value.")
+        
+    @property
+    def n_bits(self):
+        "Access to the number of bits of the binary value."
+        return self._get_n_bits()       
```

Comment:
Can you just inline this method, since it's private?

Reply:
Done